test/fixtures/gcs-fixture/src/main/java/fixture/gcs/GoogleCloudStorageHttpHandler.java

```diff
@@ -225,8 +225,13 @@ public void handle(final HttpExchange exchange) throws IOException {
                 final int start = getContentRangeStart(range);
                 final int end = getContentRangeEnd(range);
 
-                final ByteArrayOutputStream out = new ByteArrayOutputStream();
-                long bytesRead = Streams.copy(wrappedRequest, out);
+                final ByteArrayOutputStream out = new ByteArrayOutputStream() {
+                    @Override
+                    public byte[] toByteArray() {
+                        return buf;
+                    }
+                };
+                long bytesRead = Streams.copy(wrappedRequest, out, new byte[128]);
```

Comment:
Probably irrelevant but it hurts my feelings to use 8k here when its just a buffered stream for the input here and many blobs that hit this aren't even 8k :)